agent/structs/config_entry_routes.go

```diff
@@ -419,6 +420,9 @@ type HTTPQueryMatch struct {
 type HTTPFilters struct {
 	Headers    []HTTPHeaderFilter
 	URLRewrite *URLRewrite
+	//TODO @sarahalsmiller this name scheme is likely to change before the release
+	Retry   *HTTPRouteRetry
+	Timeout *HTTPRouteTimeout
```

Comment:
Seeing `HTTPHeaderFilter` above also makes me lean towards `...RetryFilter` and `...TimeoutFilter`

Reply:
Updated